dashboards/Grafana-tracequality.json

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
Dashboard creation can be templated through the mixin (we can use it for future releases if dashboards get bigger) 
Ref- https://github.com/jaegertracing/jaeger/pull/1668

Reply:
+1 I think we could have more dashboards.

A dashboard I am thinking about would show other derived metrics from traces per service. We could also define alerts for these metrics.

Reply:
I'm not sure if users would need alerts on these metrics, but definitely +1 for dashboards.